hw2/05-form.html

```diff
@@ -7,13 +7,93 @@
     <title>05 - Form</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="05-form.js"></script>
   </head>
-  <body>
-    <main>
-      <form>
-        <h1>Exercise 05 - Form</h1>
+  <body class="bg-dark">
+    <main class="container my-5 p-3 text-left w-50 bg-light rounded">
+      <form id="regForm" action="submit">
+        <h1 class="text-center mb-5">Exercise 05 - Form</h1>
+        <div class="mb-3 form-group">
+          <label class="form-label" for="name">Full Name:</label>
+          <input class="form-control" type="text" id="name" name="name" />
+        </div>
+        <div class="mb-3 form-group">
+          <label class="form-label" for="email">Email:</label>
+          <input class="form-control" type="email" id="email" name="email" />
+        </div>
+        <div class="mb-3 form-group">
+          <label class="form-label" for="regStatus">Regitration Status:</label>
+          <select
+            class="form-select"
+            name="statuses"
+            id="regStatus"
+            name="regStatus"
+          >
+            <option selected value="">Choose an Option</option>
+            <option value="Registered">Registered</option>
+            <option value="Unregistered">Unregistered</option>
+          </select>
+        </div>
+        <fieldset class="mb-3 form-group">
+          <legend class="form-label">
+            Have you taken any of the following courses?
+          </legend>
+          <div class="form-check">
+            <input
+              class="form-check-input"
+              type="checkbox"
+              value="Programming Languages"
+              id="progLang"
+              name="course"
+            />
+            <label class="form-check-label" for="progLang"
+              >Programming Languages</label
+            >
+          </div>
+          <div class="form-check">
+            <input
+              class="form-check-input"
+              type="checkbox"
+              value="Operating Systems"
+              id="opSys"
+              name="course"
+            />
+            <label class="form-check-label" for="opSys"
+              >Operating Systems</label
+            >
+          </div>
+          <div class="form-check">
+            <input
+              class="form-check-input"
+              type="checkbox"
+              value="Full Stack Web Development"
+              id="fullstack"
+              name="course"
+            />
+            <label class="form-check-label" for="fullStack"
```

Comment:
This checkbox is giving an accessibility error. There's a spelling mismatch in for and id attribute in fullstack!

Reply:
Thank you! I completely missed this.